### PR TITLE
✨ Open pdf in browser

### PR DESCRIPTION
**Pull request by williamchong**
image

### CODE REVIEW

File: src/components/NFTPage/PreviewCard.vue [hunk 1]
```diff
@@ -284,7 +284,14 @@ export default {
     handleClickViewContent() {
       this.$emit('view-content');
     },
-    handleClickViewContentUrls(type) {
+    handleClickViewContentUrls(e, contentUrl) {
+      const type = this.getContentUrlType(contentUrl);
```
Comment (nwingt) [inline src/components/NFTPage/PreviewCard.vue:288]: Besides `sameAs` field, do we need to handle if the content URL is a PDF too?

Comment (williamchong) [reply src/components/NFTPage/PreviewCard.vue:288]: UI only shows the "PDF" button if it's in sameAs
Also we dont really know if it's a pdf unless url explicitly end with .pdf
But we can add this for content url too if needed

File: src/pages/reader.vue [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="h-screen">
```
Comment (nwingt) [inline src/pages/reader.vue:2]: `h-screen`(100vh) behavior is strange in iOS, should hack Tailwind CSS to use 100dvh with 100vh as fallback

https://github.com/likecoin/liker-land/assets/17264716/194ef198-43a0-4c78-ba0a-6412c692b9e8

https://stackoverflow.com/questions/75079019/tailwind-css-fallback-for-new-screen-length-types-such-as-lvh-svh

**Review by nwingt — APPROVED** [review]
Tested OK


https://github.com/likecoin/liker-land/assets/17264716/a458a683-3029-45f4-8dba-bc12bb8901e2

**Pull request merged**